drivers/SmartThings/philips-hue/src/disco.lua

```diff
@@ -63,26 +63,26 @@ function HueDiscovery.search_for_bridges(driver, computed_mac_addresses, callbac
   local mdns_responses, err = mdns.discover(SERVICE_TYPE, DOMAIN)
 
   if err ~= nil then
-    log.error("Error during service discovery: ", err)
+    log.error_with({ hub_logs = true }, "Error during service discovery: ", err)
     return
   end
 
   if not (mdns_responses and mdns_responses.found and #mdns_responses.found > 0) then
-    log.debug("No mdns responses for Hue service this attempt, continuing...")
+    log.debug_with({ hub_logs = true }, "No mdns responses for Hue service this attempt, continuing...")
     return
   end
 
   for _, info in ipairs(mdns_responses.found) do
     if not net_utils.validate_ipv4_string(info.host_info.address) then -- we only care about the ipV4 types here.
-      log.debug("Invalid IPv4 address: " .. info.host_info.address)
+      log.debug_with({ hub_logs = true }, "Invalid IPv4 address: " .. info.host_info.address)
       goto continue
     end
     if info.service_info.service_type ~= SERVICE_TYPE then -- response for a different service type. Shouldn't happen.
-      log.debug("Unexpected service type response: " .. info.service_info.service_type)
+      log.debug_with({ hub_logs = true }, "Unexpected service type response: " .. info.service_info.service_type)
       goto continue
     end
     if info.service_info.domain ~= DOMAIN then -- response for a different domain. Shouldn't happen.
-      log.debug("Unexpected domain response: " .. info.service_info.domain)
+      log.debug_with({ hub_logs = true }, "Unexpected domain response: " .. info.service_info.domain)
```

Comment:
I could see these wanting to be info level to diagnose reasons for devices not joining. Up to you though.

Reply:
I wanted to leave these at Debug because they can be potentially chatty and we're likely not debugging an mDNS issue.